test/e2e/tests/test_table.py

```diff
@@ -155,3 +267,391 @@ def test_enable_ttl(self, forum_table):
         assert ttl["AttributeName"] == "ForumName"
         ttl_status = ttl["TimeToLiveStatus"]
         assert ttl_status in ("ENABLED", "ENABLING")
+
+    def test_enable_stream_specification(self, forum_table):
+        (ref, res) = forum_table
+
+        table_name = res["spec"]["tableName"]
+
+        # Check DynamoDB Table exists
+        assert self.table_exists(table_name)
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Disable stream
+        cr["spec"]["streamSpecification"] = {
+            "streamEnabled": False,
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.stream_specification_matches(False),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Disable stream
+        cr["spec"]["streamSpecification"] = {
+            "streamEnabled": True,
+            "streamViewType": "NEW_AND_OLD_IMAGES"
+        }
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.stream_specification_matches(True),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
+    def test_update_billing_mode(self, billing_table):
+        (ref, res) = billing_table
+
+        table_name = res["spec"]["tableName"]
+
+        # Check DynamoDB Table exists
+        assert self.table_exists(table_name)
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Set billing mode to PROVISIONED
+        cr["spec"] = {
+            "billingMode": "PROVISIONED",
+            "provisionedThroughput": {
+                "readCapacityUnits": 5,
+                "writeCapacityUnits": 5
+            }
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.billing_mode_matcher("PROVISIONED"),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
+        # Need more billing mode updates quota (only from PROVISIONED -> PAY_PER_REQUEST)
+        """ # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Disable stream
+        cr["spec"] = {
+            "billingMode": "PAY_PER_REQUEST",
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.billing_mode_matcher("PAY_PER_REQUEST"),
+            timeout_seconds=60*3,
+            interval_seconds=5,
+        ) """
```

Comment:
Add `assert`ions here to confirm that previously-set read/write capacities no longer present in `ProvisonedThroughput`

Reply:
The wait function already does that. It will fail if a field doesn't reach the intended state

Reply:
See comment above